compose.override.dist.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.8"
+services:
+  php:
+    image: piotrkardasz/sylius-php:8.2-fixuid-alpine
```

Comment:
1. I recommend considering whether we put Dockerfiles directly in the repo, so the developer can override it **or** create a `Dockerfile` inside the repository and extend your Dockerfile in it.
2. Did you consider using FrankenPHP? It's pretty cool (I base my images on it) and might bring us a marketing value. Kevin is working on it hard, and they have pretty good docs, also describing how to use FrankenPHP in production.

Reply:
`Dockerfile` is definitely a must for me for `Sylius-Standard`, as end-user should be able to adjust his/her dev environment.

Reply:
1. I would disagree, because you don't need to be an export to hijack the service configuration. If you are advanced in this topic you can always create your own dockerfile and tune the base configuration or even create a dedicated one. (The trick with an image is that you can quickly inject another image in the fly e.g. with xdebug to avoid wasting time on rebuilding the layer of the image and run the project in minutes). 
3. Yes, I have tested the pre-stable version and now I know there is a stable release. We can create an alternative configuration if needed, but in my opinion, I would go with the services configuration that everyone knows. I want to keep the configuration simple to create a Swiss army knife.

Reply:
I'd be in favor of going full into FrankenPHP.
- there's one container (frankenphp) instead of two (nginx + php-fpm)
- it's simpler to use it
- it's production ready with pretty good documentation and examples
- it seems the community is going to adapt it, and it'd be cool to help FrankenPHP grow

Reply:
FrankenPHP is nice to have 🚀 but IMHO not needed for typical Sylius development. I would stay with the current level of complexity to not add another dependency that is still not well known and requires additional understanding of how HTTP requests are handled + additional Symfony Runtime setup is needed.
If we really want to have FrankenPHP onboard Sylius, I would prefer to have an optional configuration or a guide on how to set it up with Sylius.

Reply:
> If we really want to have FrankenPHP onboard Sylius, I would prefer to have an optional configuration or a guide on how to set it up with Sylius.
Np, we can add it in our new docs we're working on.